src/openklant/components/klanten/models/klanten.py

```diff
@@ -88,10 +97,19 @@ class Klant(APIMixin, models.Model):
         default=False, help_text=_("Geeft aan of de KLANT wel of niet geverifieerd is.")
     )
 
+    objects = KlantManager()
+
     class Meta:
         verbose_name = "klant"
         verbose_name_plural = "klanten"
-        unique_together = ("bronorganisatie", "klantnummer")
```

Comment:
I don't quite understand why you remove the DB constraint here
Anyway it looks like the `makemigrations` should be run here

Reply:
This is because unique_together in DRF makes both fields implicitly required, also in the serializer (so you can't create a klant without having both fields specified):

https://www.django-rest-framework.org/api-guide/validators/#uniquetogethervalidator

> Note: The UniqueTogetherValidator class always imposes an implicit constraint that all the fields it applies to are always treated as required. Fields with default values are an exception to this as they always supply a value even when omitted from user input.

I noticed that this validation was done very early when performing a create, sidestepping the fact that I made klantnummer optional in the serializer. Alternative ideas are welcome, as I would preferably keep the unique_together on a DB level.

Reply:
In Open Zaak we made a custom validator which replaces the default one. As far as I understand the default validators is used only if you don't have `Meta.validators` in your serializer. But it's more work of course :)

Reply:
migration added